src/plugins/dialog/math.js

```diff
@@ -0,0 +1,280 @@
+                    <p style='font-size:13px;' class='_se_math_preview'></p>
+                </div>
+            </div>
+            <div class='se-dialog-footer'>
+                <button type='submit' class='se-btn-primary' title='${lang.dialogBox.submitButton}'><span>${lang.dialogBox.submitButton}</span></button>
+            </div>
+        </form>
+        `;
+
+        return dialog;
+    },
+
+    /** modify controller button */
+    setController_MathButton: function () {
+        const lang = this.lang;
+        const math_btn = this.util.createElement('DIV');
+
+        math_btn.className = 'se-controller se-controller-link';
+        math_btn.innerHTML = `
+        <div class='se-arrow se-arrow-up'></div>
+        <div class='link-content'>
+            <div class='se-btn-group'>
+                <button type='button' data-command='update' tabindex='-1' class='se-tooltip'>
+                    <i class='se-icon-edit'></i>
+                    <span class='se-tooltip-inner'><span class='se-tooltip-text'>${lang.controller.edit}</span></span>
+                </button>
+                <button type='button' data-command='delete' tabindex='-1' class='se-tooltip'>
+                    <i class='se-icon-delete'></i>
+                    <span class='se-tooltip-inner'><span class='se-tooltip-text'>${lang.controller.remove}</span></span>
+                </button>
+            </div>
+        </div>
+        `;
+
+        return math_btn;
+    },
+
+    submit: function (e) {
+        this.showLoading();
+
+        e.preventDefault();
+        e.stopPropagation();
+
+        const submitAction = function () {
+            if (this.context.math.focusElement.value.trim().length === 0) return false;
+
+            const contextMath = this.context.math;
+            const mathExp = contextMath.focusElement.value;
+            const katexEl = new this._w.DOMParser()
+                .parseFromString(contextMath.previewElement.innerHTML, 'text/html')
+                .querySelector('.katex');
+            katexEl.setAttribute('data-exp', mathExp);
+            katexEl.setAttribute('data-font-size', contextMath.fontSizeElement.value);
+            katexEl.style.fontSize = contextMath.fontSizeElement.value;
+
+            if (!this.context.dialog.updateModal) {
+                const selectedFormats = this.getSelectedElements();
+
+                if (selectedFormats.length > 1) {
+                    const oFormat = this.util.createElement(selectedFormats[0].nodeName);
+                    oFormat.appendChild(katexEl);
+                    this.insertNode(oFormat);
+                } else {
+                    this.insertNode(katexEl);
+                }
+
+                const empty = this.util.createTextNode(this.util.zeroWidthSpace);
+                katexEl.parentNode.insertBefore(empty, katexEl.nextSibling);
+            } else {
+                const findParent = (child, className) => {
```

Comment:
Arrow functions are not supported in IE11.
```javascript
function (child, className) { ...
```

Reply:
done